blocks/ds-page-navigation/readme.txt

```diff
@@ -0,0 +1,55 @@
+=== Ds Page Navigation ===
```

Comment:
Nice block specific readme

Reply:
Thanks but I don't take no credit for the Readme, those are autogenerated when using the npx commands